src/app/courses-page/pipes/order-by.pipe.ts

```diff
@@ -0,0 +1,12 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'orderBy'
```

Comment:
here and for all other directives and pipes it will be a good practice to use application specific prefix